prompting/tools/datasets/wiki.py

```diff
@@ -208,13 +208,18 @@ def get(
         )
         if not sections:
             return None
-
-        key = header, section_title = selector(list(sections.keys()))
-        content = "\n".join(sections[key])
-        section_length = len(content.split())
+        if selector == 'all':
+            content = "\n".join(["\n".join(section) for section in sections.values()])
+            section_length = len(content.split())
+            topic = "All Sections"
```

Comment:
Minor suggestion to be consistent with the quotes style across the project, in our case `"`